resources/nats/templates/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "nats.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "nats.labels" . | nindent 4 }}
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      {{- include "nats.selectorLabels" . | nindent 6 }}
```

Comment:
Same https://github.com/kyma-project/nats-manager/pull/99/files#r1295830312.

Reply:
it is like this in other files, so lets keep it like this?